libobs/graphics/device-exports.h

```diff
@@ -45,68 +45,57 @@ EXPORT gs_texture_t *device_texture_create(gs_device_t *device, uint32_t width,
 		uint32_t levels, const uint8_t **data, uint32_t flags);
-EXPORT gs_texture_t *device_voltexture_create(gs_device_t *device,
-		uint32_t width, uint32_t height, uint32_t depth,
-		enum gs_color_format color_format, uint32_t levels,
-		const uint8_t **data, uint32_t flags);
+EXPORT gs_texture_t *device_voltexture_create(gs_device_t *device, uint32_t width,
+		uint32_t height, uint32_t depth, enum gs_color_format color_format,
+		uint32_t levels, const uint8_t **data, uint32_t flags);
 EXPORT gs_zstencil_t *device_zstencil_create(gs_device_t *device,
-		uint32_t width, uint32_t height,
-		enum gs_zstencil_format format);
+		uint32_t width, uint32_t height, enum gs_zstencil_format format);
 EXPORT gs_stagesurf_t *device_stagesurface_create(gs_device_t *device,
 		uint32_t width, uint32_t height,
 		enum gs_color_format color_format);
-EXPORT gs_samplerstate_t *device_samplerstate_create(gs_device_t *device,
-		const struct gs_sampler_info *info);
+EXPORT gs_samplerstate_t *device_samplerstate_create(
+		gs_device_t *device, const struct gs_sampler_info *info);
 EXPORT gs_shader_t *device_vertexshader_create(gs_device_t *device,
-		const char *shader, const char *file,
-		char **error_string);
+		const char *shader, const char *file, char **error_string);
 EXPORT gs_shader_t *device_pixelshader_create(gs_device_t *device,
-		const char *shader, const char *file,
-		char **error_string);
-EXPORT gs_vertbuffer_t *device_vertexbuffer_create(gs_device_t *device,
-		struct gs_vb_data *data, uint32_t flags);
-EXPORT gs_indexbuffer_t *device_indexbuffer_create(gs_device_t *device,
-		enum gs_index_type type, void *indices, size_t num,
-		uint32_t flags);
-EXPORT enum gs_texture_type device_get_texture_type(
-		const gs_texture_t *texture);
-EXPORT void device_load_vertexbuffer(gs_device_t *device,
-		gs_vertbuffer_t *vertbuffer);
-EXPORT void device_load_indexbuffer(gs_device_t *device,
-		gs_indexbuffer_t *indexbuffer);
-EXPORT void device_load_texture(gs_device_t *device, gs_texture_t *tex,
-		int unit);
-EXPORT void device_load_samplerstate(gs_device_t *device,
-		gs_samplerstate_t *samplerstate, int unit);
-EXPORT void device_load_vertexshader(gs_device_t *device,
-		gs_shader_t *vertshader);
-EXPORT void device_load_pixelshader(gs_device_t *device,
-		gs_shader_t *pixelshader);
-EXPORT void device_load_default_samplerstate(gs_device_t *device, bool b_3d,
-		int unit);
+		const char *shader, const char *file, char **error_string);
+EXPORT gs_vertbuffer_t *device_vertexbuffer_create(
+		gs_device_t *device, struct gs_vb_data *data, uint32_t flags);
+EXPORT gs_indexbuffer_t *   device_indexbuffer_create(gs_device_t *device,
+		   enum gs_index_type type, void *indices, size_t num,
+		   uint32_t flags);
+EXPORT enum gs_texture_type device_get_texture_type(const gs_texture_t *texture);
+EXPORT void                 device_load_vertexbuffer(
+				gs_device_t *device, gs_vertbuffer_t *vertbuffer);
+EXPORT void device_load_indexbuffer(
```

Comment:
Alignment lost because of incorrect wrapping.

Reply:
Probably will disable all alignment options due to this, they don't seem to add to much to the actual readability of the code.